.github/workflows/pytest.yaml

```
@@ -17,6 +17,12 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install Poetry
       uses: abatilo/actions-poetry@v2.0.0
+    - name: apt update
+      run: sudo apt-get update
+    - name: Install blas for Numpy from source
+      run: sudo apt-get install -y libblas-dev #highly recommended for numpy from source, check https://numpy.org/devdocs/user/building.html#prerequisites
+    - name: Install lapack for Numpy from source
+      run: sudo apt-get install -y liblapack-dev #highly recommended for numpy from source, check https://numpy.org/devdocs/user/building.html#prerequisites
```

You can do that in one single step:
```suggestion
    - name: Install lapack/blas for Numpy from source
      run: sudo apt-get update && sudo apt-get install -y liblapack-dev libblas-dev #highly recommended for numpy from source, check https://numpy.org/devdocs/user/building.html#prerequisites
```

---

right. done